### PR TITLE
Add CDB to PyMechanical example

### LINKED ISSUE #702: Add CDB to PyMechanical example
Add example which imports CDB + ACPCompositeDefinitions.h5 into Mechanical.

### PR DESCRIPTION
**Pull request by greschd**
Add an example which loads the CDB file and ACPCompositeDefinitions.h5 into PyMechanical (after running PyACP), and then runs the analysis inside PyMechanical. PyDPF Composites is used for post-processing.

Closes #702.

### CODE REVIEW

File: examples/workflows/06-cdb-to-pymechanical-workflow.py [hunk 1]
```diff
@@ -0,0 +1,380 @@
...
+
+
+# %%
+# Import mesh, materials and plies into Mechanical
+# ------------------------------------------------
+#
+# Import geometry, mesh, and named selections into Mechanical
+
+
+# TODO: rename 'import_acp_solid_mesh'? Here it is used with a shell mesh.
```
Comment (greschd) [inline examples/workflows/06-cdb-to-pymechanical-workflow.py:281]: To discuss.

Comment (roosre) [reply examples/workflows/06-cdb-to-pymechanical-workflow.py:206]: Yes, the function load a CDB into Mechanical and sets some key options specific to layered elements. 
what about `import_acp_mesh_from_cdb`?

File: src/ansys/acp/core/mechanical_integration_helpers.py [hunk 2]
```diff
@@ -87,8 +87,8 @@ def import_acp_solid_mesh(*, mechanical: "pymechanical.Mechanical", cdb_path: PA
     """
     cdb_path = pathlib.Path(cdb_path)
 
-    if cdb_path.suffix != ".cdb":
-        raise ValueError(f"The CDB file extension must be '.cdb', not '{cdb_path.suffix}'.")
+    # if cdb_path.suffix != ".cdb":
```
Comment (roosre) [inline src/ansys/acp/core/mechanical_integration_helpers.py:90]: Why is that commented out? Or can it just be removed?

**Review by roosre — APPROVED** [review]

File: examples/workflows/06-cdb-to-pymechanical-workflow.py (hunk 1 above)
Comment (greschd) [reply examples/workflows/06-cdb-to-pymechanical-workflow.py:206]: How about `import_layered_element_mesh`? In this example we are actually loading the original `.dat` file that was also loaded into ACP.
I guess we _could_ instead use the CDB exported from ACP, not sure if that's supposed to make a difference.

File: src/ansys/acp/core/mechanical_integration_helpers.py (hunk 2 above)
Comment (greschd) [reply src/ansys/acp/core/mechanical_integration_helpers.py:90]: FMPOV, it should be extended to also allow `.dat`.

File: examples/workflows/06-cdb-to-pymechanical-workflow.py (hunk 1 above)
Comment (roosre) [reply examples/workflows/06-cdb-to-pymechanical-workflow.py]: I would limit it to CDBs (cdb, dat) generated by ACP only for the moment. Other formats might be added in the future such as k (LS Dyna input file) but the current helper function contains some CDB specific options such as "check valid block format" which we disable.

**Pull request merged**